runtime/bindings/python/initialize_module.cc

```diff
@@ -16,6 +16,13 @@
 #include "iree/base/internal/flags.h"
 #include "iree/hal/drivers/init.h"
 
+namespace {
+// Stable storage for flag processing.  Flag handling uses string views,
+// expecting the caller to keep the original strings around for as long
+// as the flags are in use.
+std::vector<std::string> alloced_flags;
```

Comment:
To make this actually-actually safe, I think you need to make this a `std::vector<std::vector<std::string>*>`. Then every call to `parse_flags` will create a new vector, add it to the global list and keep the strings forever. Otherwise, we have no way to know with repeated calls that a string from a prior call isn't still in use somewhere.

Reply:
Yes, you're right!  I knew that, but was only going for actually safe, not actually-actually safe!  :-D Having chased these strings down to the bottom, they're done with once the flags have been consumed.  That could change at anytime in the future, of course, so I will implement this fix-fix.

Reply:
Done.  Now safely safey-safe.